### PR TITLE
[DOCFIX] Update docs

### PR DESCRIPTION
**Pull request by aaudiber**
Followup to https://github.com/Alluxio/alluxio/pull/3890

This was tedious and error-prone, hopefully we can move all of this to java some day and autogenerate these files.

### CODE REVIEW

File: docs/_data/table/master-configuration.csv [hunk 1]
```diff
@@ -4,7 +4,7 @@ alluxio.master.heartbeat.interval.ms,1000
 alluxio.master.hostname,localhost
 alluxio.master.file.async.persist.handler,alluxio.master.file.async.DefaultAsyncPersistHandler
 alluxio.master.format.file_prefix,""_format_""
-alluxio.master.journal.folder,${alluxio.home}/journal/
+alluxio.master.journal.folder,${alluxio.work.directory}/journal
```
Comment (calvinjia) [inline docs/_data/table/master-configuration.csv]: Unfortunately we use `folder` here? Is the goal to switch to `directory`?

Comment (aaudiber) [inline docs/_data/table/master-configuration.csv]: We're trying to switch to directory wherever possible to be consistent. We could add an alluxio.master.journal.directory parameter with default ${alluxio.master.journal.folder} and replace the docs for .folder with .directory - if that sounds good to you I will do it in a followup PR

Comment (calvinjia) [inline docs/_data/table/master-configuration.csv]: That works for me.

File: docs/_data/table/common-configuration.csv [hunk 2]
```diff
@@ -1,19 +1,20 @@
 propertyName,defaultValue
+alluxio.conf.directory,${alluxio.home}/conf
 alluxio.debug,false
 alluxio.home,/mnt/alluxio_default_home
 alluxio.logs.dir,${alluxio.home}/logs
 alluxio.keyvalue.enabled,false
 alluxio.keyvalue.partition.size.bytes.max,512MB
-alluxio.metrics.conf.file,${alluxio.home}/conf/metrics.properties
+alluxio.metrics.conf.file,${alluxio.conf.directory}/metrics.properties
```
Comment (calvinjia) [inline docs/_data/table/common-configuration.csv]: When do we use `directory` vs `dir`?

Comment (aaudiber) [inline docs/_data/table/common-configuration.csv]: previously we used only `dir`, so I've switched to using `dir` for the new configuration properties

File: docs/_data/table/en/common-configuration.yml [hunk 3]
```diff
@@ -25,7 +27,7 @@ alluxio.site.conf.dir:
 alluxio.test.mode:
   Flag used only during tests to allow special behavior.
 alluxio.underfs.address:
-  Alluxio folder in the underlayer file system.
+  Alluxio directory in the underlayer file system.
```
Comment (gpang) [inline docs/_data/table/en/common-configuration.yml]: NIT: underlayer -> under

**Comment by gpang** [comment]
@aaudiber I just had a minor comment, but looks good.

File: docs/_data/table/en/common-configuration.yml (hunk 3 above)
Comment (aaudiber) [inline docs/_data/table/en/common-configuration.yml]: updated

**Comment by aaudiber** [comment]
@calvinjia @gpang All comments address, PTAL

**Comment by gpang** [comment]
LGTM

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**